package.json

```diff
@@ -12,21 +12,25 @@
     "watch": "gulp watch",
     "test": "gulp test",
     "tdd": "gulp tdd",
-    "changelog": "gulp changelog"
+    "changelog": "gulp changelog",
+    "release": "gulp release:minor",
+    "release:patch": "gulp release:patch",
+    "release:minor": "gulp release:minor",
+    "release:major": "gulp release:major"
   },
   "keywords": [
     "rollup",
     "rollup-plugin"
   ],
   "author": "Mickael Jeanroy <mickael.jeanroy@gmail.com>",
```

Comment:
I think we should be putting our name on these. Go with `VertiGIS` as the author and put `vertigis` back in the URLs. 

We need to credit the author, which we do in the readme, but we are publishing this fork of the package.

Reply:
Done